iroha/src/torii.rs

```diff
@@ -254,13 +254,20 @@ async fn handle_queries(
     Ok(result)
 }
 
+#[derive(Deserialize, Serialize, Clone)]
+struct StatusHealth {
+    status: Health,
+}
```

Comment:
Why do we need a wrapper around Health? We can just return a properly formatted enum to json to my mind.

Also even though you use `Json(StatusHealth)`, you still implement HttpResponse for Health, I am not sure why?

Reply:
Then it will return bare string. I thought that json object would be prettier

Reply:
You can use adjacently tagged repr https://serde.rs/enum-representations.html#adjacently-tagged